data/index.html

```diff
@@ -146,26 +146,34 @@ <h2>SGID Index</h2>
                 </a>
             </div>
-            <p class="readable">The SGID Index is a sortable and searchable table of GIS resources from AGRC and many other state agencies. It is a great resource to quickly find GIS data and resources important to you. The index lists the resource's ISO category, name, and primary data steward, along with a short description of the resource and a link to a service endpoint (if available). </p>
+            <p class="readable">The SGID Index is the master source of information about SGID datasets from AGRC and other GIS resources from many other state agencies. It is a great resource to quickly find GIS information important to you. The index lists the resource's ISO category, name, and primary data steward, along with a short description of the resource and a link to a service endpoint (if available). </p>
             <p class="section-read-more">
                 <a href="{% link data/sgid-index/index.html %}" class="btn">Search the Index</a>
             </p>
     </div>
     <div class="grid__col section">
         <h2>The Open SGID</h2>
-        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing our curated collection of GIS datasets. It is a publicly-available database connection that can be used in many GIS platforms to directly access Utah spatial data. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
+        <p class="readable">The Open SGID is a cloud-hosted PostGIS database containing our curated collection of GIS datasets. You can use this publicly available database connection to directly access Utah spatial Data in a wide variety of applications, including most GIS software. You can find the most up-to-date information about the Open SGID, including troubleshooting tips and an issue tracker, in the <a href="https://github.com/agrc/open-sgid">GitHub repository</a>. Please read, understand, and abide by the <a href="{% link about/policy/open-sgid/index.md %}">terms of use</a> prior to connecting.</p>
         <p class="section-read-more">
             <a href="{% link sgid/open-sgid/index.md %}" class="btn">Get Connected</a>
         </p>
     </div>
     <div class="grid__col section alternate">
+        <h2>ArcGIS Online Hosted Feature Services</h2>
+        <p class="readable">AGRC also uploads all our curated datasets to <a href="https://utah.maps.arcgis.com">ArcGIS Online</a> as hosted feature services. You can search for these layers in Esri products and add them to your projects. All the SGID hosted feature services' names start with "Utah" (like "Utah Roads") and have a branded thumbnail. They are also shared with a Utah SGID group that matches their category.</p>
```

Comment:
**question**: are groups an open data concept as well as an arcgis online concept?

Reply:
They are interrelated. You share data to an Open Data site by adding it to an AGOL group that is "enabled for Open Data". When we created our site, it looks like we configured the tiles of our site to apply a search-by-group filter on our Open Data list. Within AGOL, you can view an organization's groups (without logging in, if they are public groups) and see the different items in each group.